api/type.rb

```diff
@@ -33,7 +33,11 @@ module Fields
 
       attr_reader :output # If set value will not be sent to server on sync
       attr_reader :input # If set to true value is used only on creation
-      attr_reader :url_param_only # If, true will not be send in request body
+
+      # url_param_only will not send the field in the resource body and will
+      # not attempt to read the field from the API response.
+      # NOTE - this doesn't work for nested Terraform fields
```

Comment:
```suggestion
      # NOTE - this doesn't work for nested fields
```

This affects all providers I think, they all share `gettable_properties` and `settable_properties`